lib/ramble/ramble/util/stats.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2022-2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+# https://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+# <LICENSE-MIT or https://opensource.org/licenses/MIT>, at your
+# option. This file may not be copied, modified, or distributed
+# except according to those terms.
+
+import statistics
+
+
+def decimal_places(value):
+    """Returns the number of decimal places of a value"""
+
+    val_str = str(value)
+    if '.' not in val_str:
+        return 0
+    else:
+        return len(val_str.split('.')[1])
+
+
+def max_decimal_places(list):
+    """Returns the max decimal places of a list of values"""
+
+    max = 0
+    for val in list:
+        if decimal_places(val) > max:
+            max = decimal_places(val)
+    return max
+
+
+class StatsBase(object):
```

Comment:
Given the stats classes are so self contained, this PR should probably have some unit tests for them.

Reply:
Added

Reply:
Awesome. Thanks!